Tasks/Fartotski/Fartotski_Task6.py

```diff
@@ -0,0 +1,27 @@
+sumNumber=0
+def sumItems(newlist):
+    for i in newlist:
+        if isinstance(i,list):
+            sumItems(i)
+        else:
+            global sumNumber
```

Comment:
it can be done without a global variable by means of additional function argument